app/assets/stylesheets/components/_toolbar-dropdown.scss

```diff
@@ -62,6 +62,33 @@ $dropdown-arrow-spacing: $dropdown-arrow-size / 2;
   padding: 0;
 }
 
+.app-c-toolbar-dropdown__link {
```

Comment:
As this is almost the same as `.app-c-toolbar-dropdown__button` could they both use the same css and save having to duplicate it?

Reply:
Further to this, I wonder if they could both be the same, more generic, class and then we'd not need to duplicate the event handlers.

Reply:
I did go with that originally, but I think this is a case of 'similar but different'. The 'link' class doesn't require a few of the styles for buttons, plus it adds one of its own.

Reply:
okey doke